core/build.gradle

```diff
@@ -6,7 +6,7 @@ repositories {
 }
 
 dependencies {
-    compile 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.1'
+    compile 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.0'
```

Comment:
yea :+1: but two PR's please :x: or lets not open/merge this until #26 is merged